modules/standard/Set.chpl

```diff
@@ -436,28 +453,40 @@ module Set {
       :rtype: `[] eltType`
     */
     proc const toArray(): [] eltType {
-      var result: [1.._dom.size] eltType;
+      var result: [0..#_htb.tableNumFullSlots] eltType;
```

Comment:
You should take the lock before reading tableNumFullSlots here

Reply:
Thanks!